sensorhub-driver-android/src/main/java/org/sensorhub/impl/sensor/android/video/AndroidCameraOutputVP9.java

```diff
@@ -62,15 +62,25 @@ public AndroidCameraOutputVP9(AndroidSensorsDriver parentModule, int cameraId, S
     }
 
 
+    @Override
+    protected void initOutputStructure() {
+        // create SWE Common data structure and encoding
+        VideoCamHelper fac = new VideoCamHelper();
+        // use the one of H264
+        DataStream videoStream = fac.newVideoOutputH264(getName(), imgWidth, imgHeight);
```

Comment:
Need to set VP9 in output encoding description

Reply:
Done using generic descriptor